tweet_trigger_readme.md

```diff
@@ -0,0 +1,26 @@
+tweet_trigger_readme.md
+
+Introduction
+------------
+
+Tweet_trigger is an application which collects earthquake events that come in through PDL and tweets them from the "notedalerts" account. It was designed to run in Python 3. 
+
+Installation and Dependencies
+-----------------------------
+
+Create a conda environment OR activate existing conda environment:
+    To create a new environment, type: conda create --name anyEnvName
+    To activate existing environment, type: source activate yourEnvName
+
+Install dependencies by typing:
+    pip install psycopg2
+    pip install tweepy
+
+Make sure this conda environment is activated for whatever user is running PDL. 
```

Comment:
Good point! please add onto this comment something like "This is because the PDL client runs this python code when a new event or updated event (origin product) is received via PDL."